### PR TITLE
add a table comparing hosted security scanners

### PR DESCRIPTION
**Pull request by afeld**
screen shot 2016-02-02 at 11 14 13 am

Also did some minor cleanup of the headings.

/cc @micahsaul @harrisj

### CODE REVIEW

**Comment by DavidEBest** [comment]
QuantifiedCode provides Python-specific code analysis. It doesn't do dependency monitoring, though.

**Comment by DavidEBest** [comment]
I can add QC. @afeld

**Comment by afeld** [comment]
Already done!

**Comment by DavidEBest** [comment]
I'm not sure it is correct to say that all of them use open source as the basis of their code analysis. Hakiri & QuantifiedCode don't make that claim.

Perhaps something like:

Code analysis can be run locally with the following open source tools. These tools provide results similar (and in some cases, identical) to the hosted services above.

File: _pages/security/static-analysis.md [hunk 1]
```diff
@@ -6,18 +6,30 @@ parent: Security
...
+* [**Dependency analysis**](#dependency-analysis), where the Ruby gems, Python modules, and JavaScript packages your app uses are checked against a list of known vulnerabilities.
+* [**Code analysis**](#code-analysis), in which your code is checked against a list of antipatterns.
 
 There are tools for JS, Ruby, and Python, and you are encouraged to set up this scanning early on in the development cycle to prevent unexpected delays when it's time to get your ATO.
 
-## Dependency Analysis (All Languages)
+### Services
 
-[Gemnasium](https://gemnasium.com) provides dependency scanning for all our supported languages – you should be able to add a public repository by clicking "add new project" after signing up/in with GitHub. If you need scanning on a private repository, [file an issue in the devops repo](https://github.com/18F/DevOps/issues/new).
+Service | Langues/frameworks | Dependency analysis | Code analysis
```
Comment (DavidEBest) [inline _pages/security/static-analysis.md]: Typo in `Languages`

**Comment by afeld** [comment]
Updated!

**Comment by DavidEBest** [comment]
:+1: @micahsaul, @harrisj any thoughts?

**Comment by micahsaul** [comment]
Looks good to me.

**Pull request merged**